src/SignalR/clients/csharp/Client.SourceGenerator/src/gen/Microsoft.AspNetCore.SignalR.Client.SourceGenerator.csproj

```diff
@@ -1,19 +1,25 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
+    <Description>Source Generator for ASP.NET Core SignalR Client</Description>
     <TargetFramework>netstandard2.0</TargetFramework>
-    <LangVersion>9</LangVersion>
     <IncludeBuildOutput>false</IncludeBuildOutput>
-    <SuppressDependenciesWhenPacking>true</SuppressDependenciesWhenPacking>
+    <SuppressDependenciesWhenPacking>false</SuppressDependenciesWhenPacking>
     <Nullable>enable</Nullable>
   </PropertyGroup>
 
   <ItemGroup>
     <Reference Include="Microsoft.CodeAnalysis.CSharp" PrivateAssets="all" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Reference Include="HubProxyAttributes" />
+  </ItemGroup>
+
   <ItemGroup>
     <None Include="$(OutputPath)\$(AssemblyName).dll" Pack="true" PackagePath="analyzers/dotnet/cs" Visible="false" />
+    <!-- The source generator includes some attributes to be used by user code to influence the source generated code -->
+    <None Include="$(OutputPath)\HubProxyAttributes.dll" Pack="true" PackagePath="lib/netstandard2.0" Visible="false" />
```

Comment:
Clever. Can we call this Microsoft.AspNetCore.SignalR.Client.HubProxyAttributes.dll or something like that?